lib/command/src/Obelisk/Command/Deploy.hs

```diff
@@ -122,6 +121,14 @@ deployPush deployPath getNixBuilders = do
     callProcess'
       (Map.fromList [("NIX_SSHOPTS", unwords sshOpts)])
       "nix-copy-closure" ["-v", "--to", "root@" <> host, "--gzip", outputPath]
+  withSpinner "Uploading config" $ ifor_ buildOutputByHost $ \host _ -> do
+    callProcessAndLogOutput (Notice, Warning) $
+      proc "rsync"
+        [ "-e ssh " <> unwords sshOpts
+        , "-qarvz"
```

Comment:
quiet & verbose? :P let's use the long option names to make it easier to read when coming back to this. This should also have `--delete` to ensure removed configs in the deploy directory get removed in the deployment targets